core/src/main/java/org/apache/accumulo/fate/FateTxId.java

```diff
@@ -54,7 +56,9 @@ public static long fromString(String fmtTid) {
    */
   public static String formatTid(long tid) {
     // do not change how this formats without considering implications for persistence
-    return String.format("%s%016x%s", PREFIX, tid, SUFFIX);
+    // original format: String.format("%s%016x%s", PREFIX, tid, SUFFIX);
+    // Since 2.1, this format was replaced with the faster version below
+    return FastFormat.toHexString(PREFIX, tid, SUFFIX);
```

Comment:
Is this a change in format, or just a change in the method called to do the formatting? If it's the former, I'm curious what's different in the new format. If it's the latter, I'm not sure this comment about the original vs. the replacement implementation is very helpful, and it might be misleading.

Reply:
> I'm not sure this comment about the original vs. the replacement implementation is very helpful, and it might be misleading.

My goal was to keep the same format just improve performance.  Assuming I didn't screw it up, then I think you are right that the comment could be misleading.